docs/multiple-users.md

```diff
@@ -0,0 +1,53 @@
+# Multiple users in GCM
+
+We're sometimes asked, "Does GCM support multiple users?" The answer is a bit complex (though ultimately, it's "yes").
```

Comment:
Do we need to re-introduce "GCM = Git Credential Manager" here for the first time in the doc?

Reply:
Yeah, that would be proper form. It'll be awkward in this pseudo-quote, though, so I need to rework that.